e2e/specs/modals.spec.js

```diff
@@ -97,6 +97,19 @@ describe("modals", () => {
     );
   });
 
+  it("renders the video recorded dialog width correctly", () => {
+    // Set the viewport to 1280 x 720 so checking for 80vw cleaner
+    cy.viewport(1280, 720)
+    cy.get("#MainMenu").click();
+    cy.get('[data-testid="main-menu-list"] > ul').eq(3).click({ force: true });
+    cy.get('.ModalBody button').click({ force: true });
+
+    cy.wait(4000);
+    cy.get("#MainMenu").type("{esc}");
+
+    cy.get("div[role='dialog']").should("have.css", "width", "1024px");
+  });
```

Comment:
Not a snapshot test because it would be flaky - examples from two runs:
<img width=300 src=https://github.com/streamlit/streamlit/assets/63436329/82467ad5-492e-4fb8-bc57-f960e5981d2b /><img width=300 src=https://github.com/streamlit/streamlit/assets/63436329/53958ec3-cb98-4bbc-87c0-48e9164bb750 />